core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -99,6 +99,15 @@ message SignedKV {
   bytes previous = 3;
 }
 
+// LeafProof for a verifiable map leaf.
+message LeafProof {
```

Comment:
This is great! 
FYI, we'll trying to move towards using Trillian.MapLeafInclusion here. 
Up to you whether you want to do this in this pull request or save it for later.